clearwater-config-manager.root/usr/share/clearwater/clearwater-config-manager/scripts/validate_json.py

```diff
@@ -101,3 +101,29 @@
 
     print(yaml.dump(temp_dict, default_flow_style=False).replace("u'", "'"))
     sys.exit(1)
+
+
+# Carry out final check, that no duplicated keys are present.
```

Comment:
I think we should do this check after we check the file is valid json, but before validating against a schema. Otherwise a duplicate key error could manifest as a schema error, which would make it harder to track down.

Reply:
I don't think this is right? The schema validation can't pick up these errors.
I've moved it anyways since I don't reckon it makes too much of a difference which errors we pick up first.

Reply:
My point was that if I had an object like

```
{
    "count": 100, // I added this one
    ...
    "count": "not an integer"
}
```

the real problem is the entry I added is being masked by the invalid one further down the object.